src/Makefile

```diff
@@ -140,6 +141,13 @@ $(SUBDIRS) : mklibdir
 $(EXT_SUBDIRS) : mklibdir
 	$(MAKE) -C $@
 
+install: all
+	mkdir -p $(INSTALLPREFIX)/{bin,lib}
+	# copying binaries to $(INSTALLPREFIX)/bin
+	find $(SUBDIRS_BIN) -type f -executable -exec cp -v {} $(INSTALLPREFIX)/bin/ ';'
+	# copying libs to $(INSTALLPREFIX)/lib
+	cp -av ../tools/openfst/lib/* $(INSTALLPREFIX)/lib/
+
```

Comment:
jtrmal, at this stage, I only installed the kaldi binaries and the libraries they depended upon (I used ldd to check all of them). If openfst binaries or other dependencies should be installed here, they can be added too, but I wasn't going to do so until our researchers complained that it wasn't working.

I think having a should-be working install target is a good start, and it can be improved later if something is missing.

My intention was that this should install to somewhere like /opt/kaldi rather than /usr so that it doesn't interfere with packages installed by the system and apt-get.